data/kucoin

```diff
@@ -0,0 +1,3 @@
+kucoin.com
+kucoin.plus
+staticimg.com
```

Comment:
Please provide evidence of this domain belonging to this category.

Reply:
No. I mean the `staticimg.com`

Reply:
![2024-07-15 06:38:00](https://github.com/user-attachments/assets/9e847554-471a-45dd-8324-572b5f786c65)

KuCoin hosts static resources on it. I searched through all the rules and didn't find any other references to `staticimg.com`, so I added it in

Reply:
> ![2024-07-15 06:38:00](https://private-user-images.githubusercontent.com/88403145/348574080-9e847554-471a-45dd-8324-572b5f786c65.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIwMDI1MzcsIm5iZiI6MTcyMjAwMjIzNywicGF0aCI6Ii84ODQwMzE0NS8zNDg1NzQwODAtOWU4NDc1NTQtNDcxYS00NWRkLTgzMjQtNTcyYjVmNzg2YzY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI2VDEzNTcxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1MTlhNmY3Y2FkN2MwYWZmYzg3MGM2NzgwMWU4MjBiOTQ5M2Q5OGY4YjlkYjk1NWY4ZDA3NTdhNmJhNWFiNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bIeXi_P9vOZpAjp09hzwpbOhheNviiEFpWS91tYfa2k)
> 
> KuCoin hosts static resources on it. I searched through all the rules and didn't find any other references to `staticimg.com`, so I added it in

No, I mean... Are you sure this domain is only use on this website?